src/main/java/com/spotinst/sdkjava/model/bl/elastigroup/aws/BeanstalkPlatformUpdate.java

```diff
@@ -0,0 +1,135 @@
+package com.spotinst.sdkjava.model.bl.elastigroup.aws;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
```

Comment:
user ignore=true and NON_NULL in all bl and api calsses

Reply:
Done